src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies?: Movie[];
```

Comment:
movies prop should be required, it can't be undefined

Reply:
fixed